VK/VK/FriendListScreen/FriendListTableViewController.swift

```diff
@@ -174,10 +136,11 @@ final class FriendListTableViewController: UITableViewController, UIViewControll
         )
     }
 
-    private func configreTapHandler() {
+    private func configureTapHandler() {
         tapHandler = { [weak self] user in
-            guard let self = self else { return }
-            guard let vc = self.storyboard?
+            guard
+                let self = self,
+                let vc = self.storyboard?
```

Comment:
```suggestion
                let vc = self.storyboard?
```
вот так вот "vc" не надо, надо вот так вот "SomeViewController"